docs/source/install.rst

```diff
@@ -233,6 +234,12 @@ the DepthAI example again inside the VM. Choose to route to VM and select to *no
 watchdog could get triggered if the host doesn't start communication in few seconds). You may need to repeat running the script a few times, until all gets 
 set properly for VMware.
 
+Virtual Box
+***********
+
+Detailed step-by-step tutorial available `here <https://docs.luxonis.com/en/latest/pages/faq/#how-to-use-depthai-under-virtualbox>`__
```

Comment:
I think it should be the other way around - we should take the contents of this FAQ section and put it here

Reply:
@VanDavv I think, we should keep that in FAQ not to break any links.

Reply:
Added in https://github.com/luxonis/depthai-docs-website/pull/193